src/main/scala/is/hail/utils/richUtils/RichCodeMemoryBuffer.scala

```diff
@@ -83,6 +82,10 @@ class RichCodeMemoryBuffer(val region: Code[MemoryBuffer]) extends AnyVal {
     region.invoke[Long, Long, Unit]("setBit", byteOff, bitOff)
   }
 
+  def setBit(byteOff: Code[Long], bitOff: Long): Code[Unit] = {
+    region.invoke[Long, Long, Unit]("setBit", byteOff, bitOff)
```

Comment:
I think this has the same effect, bitOff gets promoted to a code here.  I wanted to inline `bitSet` and constant fold: the bit shift and the mask computation should be static, so it should just be a add (with constant), load, bitor (with constant) and store.

Reply:
Although I freely admit this might be micro-optimization...

Reply:
womp, you're right. I think I might be inclined to leave this here right now as a placeholder function for if/when we remove the region.invoke... stuff for direct invocation of the static Memory methods, since I think we found that these calls to the MemoryBuffer functions don't get inlined anyways. What do you think?

Reply:
Fair enough.  Approving.